recipes/dd-opentracing-cpp/all/conanfile.py

```diff
@@ -0,0 +1,105 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class DatadogOpenTracingConan(ConanFile):
+    name = "dd-opentracing-cpp"
+    description = "Monitoring service for cloud-scale applications based on OpenTracing "
+    license = "Apache 2.0 License"
```

Comment:
spdx short identifier please
```suggestion
    license = "Apache-2.0"
```